src/blueprints/tv2/action-factories/tv2-split-screen-action-factory.ts

```diff
@@ -124,7 +130,13 @@ export class Tv2SplitScreenActionFactory extends ActionFactory {
           },
           {
             type: MutateActionType.PIECE,
-            updateActionWithPiece: (action: Action, piece: Piece) => this.updateInsertToInputAction(action, piece),
+            updateActionWithPiece: (action: Action, onAirPiece: Piece | undefined, nextPiece: Piece | undefined): Action => {
+              if (onAirPiece)
+                return this.updateInsertToInputAction(action, onAirPiece)
+              else if (nextPiece)
+                return this.updateInsertToInputAction(action, nextPiece)
+              return action
```

Comment:
```suggestion
              if (onAirPiece) return this.updateInsertToInputAction(action, onAirPiece)
              if (nextPiece) return this.updateInsertToInputAction(action, nextPiece)
              return action
```

Reply:
Removed the `else`. Kept the indentation.